pymeasure/instruments/signalrecovery/dsp7265.py

```diff
@@ -35,60 +35,76 @@ class DSP7265(Instrument):
     voltage = Instrument.control(
         "OA.", "OA. %g",
         """ A floating point property that represents the voltage
-        in Volts. This property can be set. """
+        in Volts. This property can be set. """,
+        validator=truncated_range,
+        values=[0,5]
     )
     frequency = Instrument.control(
         "OF.", "OF. %g",
         """ A floating point property that represents the lock-in
-        frequency in Hz. This property can be set. """
-    ) 
+        frequency in Hz. This property can be set. """,
+        validator=truncated_range,
+        values=[0,2.5e5]
+    )
     dac1 = Instrument.control(
         "DAC. 1", "DAC. 1 %g",
         """ A floating point property that represents the output
-        value on DAC1 in Volts. This property can be set. """
+        value on DAC1 in Volts. This property can be set. """,
+        validator=truncated_range,
+        values=[-12,12]
```

Comment:
Looking at the manual, I am not clear on whether these values should be over the range -12 to 12 V or -12000 to 12000 (fixed point mode). Can you confirm that the command "DAC. 1" is the normal floating point number? The same comments go for the remainder of the added ranges.

Reply:
Giving the command as "DAC. 1 %g" (with the period included) is floating point mode, so the lockin should expect a normal floating point number as written.

Reply:
Got it, thanks!